.stylelintrc.json

```diff
@@ -1,7 +1,38 @@
 {
   "extends": "stylelint-config-recommended",
-  "plugins": ["stylelint-order", "stylelint-selector-bem-pattern"],
+  "plugins": [
+    "./tools/stylelint/index.js",
```

Comment:
I have changed my mind from our chat about the naming of the `tools/` directory. We should place the `stylelint/` directory at the root of the repo. If we then come to a time where we need to create custom front end processing tools, then we can think about putting the code in a `tool/` like directory.

Reply:
👍